powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/ActivationServiceBehaviorTest.java

```diff
@@ -71,7 +67,7 @@ class ActivationServiceBehaviorTest {
 
     private final KeyConvertor keyConvertor = new KeyConvertor();
     private final ObjectMapper objectMapper = new ObjectMapper();
-    private final String version = "3.2";
+    private final String version = "3.1";
```

Comment:
Why don't we use version `3.2`? It is the latest version of the protocol.

Reply:
🤔 Most probably I overcomplicated that, because it was reported earlier. Let's change it to **3.2**